crates/core_simd/src/vector.rs

```diff
@@ -313,11 +313,11 @@ where
     #[inline]
     pub fn gather_select(
         slice: &[T],
-        enable: Mask<isize, LANES>,
+        enable: Mask<usize, LANES>,
```

Comment:
note x86 has mask sizes match the *data* size, not the index/address size...we should probably match that:
https://www.felixcloutier.com/x86/vgatherdps:vgatherqps#vgatherqps--vex-128-version-

Reply:
opened #323 to track this.

Reply:
Good point. With this change it would be easy to make this take `Mask<T, LANES>` instead, but I'll leave that to a separate PR.